client/deposits/list/test/__snapshots__/index.js.snap

```diff
@@ -83,7 +83,7 @@ exports[`Deposits list renders correctly 1`] = `
           <tbody>
             <tr>
               <th
-                class="woocommerce-table__header info-button"
+                class="woocommerce-table__header info-button is-sorted"
```

Comment:
That column doesn't have a title (it only shows the info icon), so it behaves like the Date column, that means that we add a grey background to both when we sort by Date (which is the default)

Without that, it would look like the first row here:
<img width="512" alt="Screen Shot 2021-03-24 at 19 30 00" src="https://user-images.githubusercontent.com/407542/112391893-5b25e700-8cd7-11eb-9d90-3ac7b4779c1d.png">